sw/device/lib/dif/dif_clkmgr.c

```diff
@@ -33,6 +34,13 @@ static dif_toggle_t bool_to_toggle(bool val) {
   return val ? kDifToggleEnabled : kDifToggleDisabled;
 }
 
+/**
+ * Converts a `multi_bit_bool_t` to `dif_toggle_t`.
+ */
+static dif_toggle_t mubi4_to_toggle(multi_bit_bool_t val) {
+  return (val == kMultiBitBool4True) ? kDifToggleEnabled : kDifToggleDisabled;
+}
+
```

Comment:
Should we inline this in `dif_base.h`? Other blocks will likely need this conversion too.

Reply:
How about we replicate the `mubi?_test_(true|false)_(strict|loose)` functions in `multibits.h`, similar to what we do in RTL?

Reply:
Yeah sure. Fine with followup discussions / amends in the interest of satisfying the CI.

Reply:
OK, split out as an issue here: #9036